src/main/java/com/upsolver/datasources/jdbc/querybuilders/DefaultQueryDialect.java

```diff
@@ -25,11 +25,12 @@ public long utcOffset(Connection connection) throws SQLException {
     public NamedPreparedStatment taskInfoByInc(TableInfo tableInfo,
                                                JDBCTaskMetadata metadata,
                                                Connection connection) throws SQLException {
-        String query = "SELECT MIN(" + tableInfo.getIncColumn() + ") AS min," +
-                " MAX(" + tableInfo.getIncColumn() + ") AS max" +
+        String incColumn = toUpperCaseIfRequired(tableInfo.getIncColumn());
+        String query = "SELECT MIN(" + incColumn + ") AS " + toUpperCaseIfRequired("min") + "," +
+                " MAX(" + incColumn + ") AS " + toUpperCaseIfRequired("max") +
                 " FROM " + fullTableName(tableInfo) +
-                " WHERE " + tableInfo.getIncColumn() + " >= :startFrom" +
-                " HAVING MIN( " + tableInfo.getIncColumn() + ") IS NOT NULL";
+                " WHERE " + incColumn + " >= :startFrom" +
```

Comment:
instead of updating the column names in the queries inside the query dialect. Which happens many times and a name can easily be missed. Update the name when creating the `TableInfo`

Reply:
done